google-login-plugin/src/com/google/gct/login/idea/PluginInitializationComponent.java

```diff
@@ -0,0 +1,34 @@
+package com.google.gct.login.idea;
+
+import com.google.gct.idea.feedback.FeedbackUtil;
+
+import com.intellij.openapi.components.ApplicationComponent;
+import com.intellij.util.PlatformUtils;
+
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Performs runtime initialization for the Google Login plugin.
+ */
+public class PluginInitializationComponent implements ApplicationComponent {
```

Comment:
Do you see value in testing the dumb getters? Google Java test practices is not to test logic-less methods.